airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java

```diff
@@ -162,13 +156,13 @@ public JsonNode toJdbcConfig(JsonNode config) {
 
       // todo - this obviously needs to actually do something.
       final Predicate<Object> hasReachedLsnPredicate = (r) -> false;
-      final Iterator<JsonNode> queueIterator = Queues.toStream(queue).iterator();
-      final AbstractIterator<JsonNode> iterator = new AbstractIterator<>() {
+      final Iterator<AirbyteMessage> queueIterator = Queues.toStream(queue).iterator();
+      final AbstractIterator<AirbyteMessage> iterator = new AbstractIterator<>() {
```

Comment:
this in-line class definition really threw me off while reading this code. Why don't we refactor it out of this method? The method is pretty large right now which makes it hard to follow.

Reply:
there's still a lot we're trying to lock down here right now. i agree, we will want to refactor this to make the responsibility of each piece more clear and testable. can we get a pass on this for now while we are still making pretty big code changes in this area?

Reply:
👍🏼